qutebrowser/config/config.py

```diff
@@ -162,13 +163,39 @@ def get_bindings_for(self, mode: str) -> Dict[keyutils.KeySequence, str]:
                 bindings[key] = binding
         return bindings
 
+    def _implied_cmd(self, cmdline: str) -> str:
+        """Return cmdline, or the implied cmd if cmdline is a set-cmd-text."""
+        try:
+            results = runners.CommandParser().parse_all(cmdline, aliases=False)
+            if len(results) == 0:
+                return ""
+            result = results[0]
+        except cmdexc.NoSuchCommandError:
```

Comment:
Please catch this directly after the `parse_all` (to have as little code as possible in the `try:` block)